tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/Emitter.scala

```diff
@@ -226,16 +232,277 @@ final class Emitter private (semantics: Semantics, outputMode: OutputMode,
       emitNextLine(0)
   }
 
+  // Private API for Rhino
+
+  private[scalajs] object rhinoAPI { // scalastyle:ignore
+    private implicit val GlobalKnowledge: GlobalKnowledge = new Invalidatable {
```

Comment:
I think this deserves a comment that you just don't want to write another set of forwarders in the `KnowledgeGuardian` (which is absolutely fair).
OTOH, if we call `Invalidatable` `KnowledgeAccessor`, this is probably much clearer.